Archiver.py

```diff
@@ -1,4 +1,30 @@
 
+import os.path
+
+
+def read_file(file_path):
+    file_path = os.path.abspath(file_path)
+    file_name = os.path.basename(file_path).split(".")[0]
+    try:
+        with open(file_path, "r") as input_file:
+            out_string = input_file.readline()
+    except FileNotFoundError:
+        print("So file is not found, try again")
+        out_string = None
+        file_name = None
+    return out_string, file_name
+
+
+def write_file(final_string, input_file_name, zipping=False):
+    if zipping:
+        output_file_name = input_file_name + "_uncompressed.txt"
```

Comment:
Can't see anything about this "_uncompressed" postfix in requirements

Reply:
Ok, what about name after unzip? It should be different from the original.